### PR TITLE
feat: use notifications v2 as the source of truth

### PR DESCRIPTION
**Pull request by idoshamun**
Read notifications from the notification v2 tables. This involves some GraphORM magic to fetch the attachments and avatars from the right place. 🪄

I also noticed we don't use the unread notifications count worker, so I removed it

See: https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/701267969/WT-1936+Optimize+notifications#Read-from-the-new-schema

Prerequisite:
- [x] https://github.com/dailydotdev/daily-api/pull/1572
- [x] Merge old data

WT-2046 #done

### CODE REVIEW

File: src/common/pubsub.ts [hunk 1]
```diff
@@ -47,7 +47,6 @@ const userDeletedTopic = pubsub.topic('user-deleted');
 const userUpdatedTopic = pubsub.topic('user-updated');
 const usernameChangedTopic = pubsub.topic('username-changed');
 const settingsUpdatedTopic = pubsub.topic('settings-updated');
-const notificationsReadTopic = pubsub.topic('api.v1.notifications-read');
```
Comment (rebelchris) [inline src/common/pubsub.ts:50]: You also removed this from streams?

File: src/schema/notifications.ts [hunk 2]
```diff
@@ -368,9 +367,9 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
           notificationsPageGenerator.nodeToCursor(page, args, node, index),
         (builder) => {
           builder.queryBuilder
-            .andWhere(`${builder.alias}."userId" = :user`, { user: ctx.userId })
-            .andWhere(`${builder.alias}."public" = true`)
-            .addOrderBy(`${builder.alias}."createdAt"`, 'DESC');
+            .andWhere(`un."userId" = :user`, { user: ctx.userId })
```
Comment (rebelchris) [inline src/schema/notifications.ts:370]: Does this scale?
Just wondering if it doesn't break

**Review by rebelchris — APPROVED** [review]
Nice nothing blocking from my side.

File: src/common/pubsub.ts (hunk 1 above)
Comment (idoshamun) [reply src/common/pubsub.ts:50]: Good point! 🙈

File: src/schema/notifications.ts (hunk 2 above)
Comment (idoshamun) [reply src/schema/notifications.ts:370]: I did everything I could to make it fast. I believe it should be ok.
Basically pagination, ordering, filtering is done on a single table `UserNotification`. It should reduce the load significantly

File: src/common/pubsub.ts (hunk 1 above)
Comment (idoshamun) [reply src/common/pubsub.ts:50]: https://github.com/dailydotdev/streams/pull/24

**Pull request merged**